src/target/riscv/riscv.c

```diff
@@ -700,6 +732,99 @@ static void log_trigger_request_info(struct trigger_request_info trig_info)
 			trig_info.tdata1, trig_info.tdata2, trig_info.tdata1_ignore_mask);
 };
 
+static struct tdata1_cache *create_new_tdata1_cache(struct list_head *tdata1_cache_head, riscv_reg_t tdata1)
```

Comment:
Following the example of e.g. riscv_batch_alloc, let's call this:
```suggestion
static struct tdata1_cache *tdata1_cache_alloc(struct list_head *tdata1_cache_head, riscv_reg_t tdata1)
```

(How does this square with e.g. riscv_free_registers()? It's a bit messy and inconsistent, but at least when there's a simple struct then using _alloc() seems more consistent.)

Reply:
Addressed.